lib/client/jwt_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "jwt"
+require "active_support"
+require "active_support/time"
+
+module Client
+  # The JwtHelper module provides a method to generate a JWT token.
+  # The token is signed using the app's id and includes an expiration time.
+  module JwtHelper
```

Comment:
This does not need to be a separate module since you are using it, in one single place only, Please merge to the client